README.md

```diff
@@ -2,21 +2,30 @@
 
 This repository contains the completed app and starter files for the [Build a Shopify app with Node and React](#) tutorial.
 
-The completed app is the master branch, and each starter file has its own branch. For example, To start the tutorial at [Build your user interface with Polaris](#) check out and use `build-your-user-interface-with-polaris-starter-files`.
+## Available branches for each step in the tutorial
+
+Each step of the [Build a Shopify app with Node and React](https://developers.shopify.com/tutorials/build-a-shopify-app-with-node-and-react) tutorial has a corresponding branch in this repository. Checkout any of these branches to view the demo project at that stage, or to use it as the starting point for your own app. For example, To start the tutorial at [Build your user interface with Polaris](#) check out and use `build-your-user-interface-with-polaris-starter-files`.
```

Comment:
looks like (https://developers.shopify.com/tutorials/build-a-shopify-app-with-node-and-react/build-your-user-interface-with-polaris) is missing for (#)

Reply:
as well as `build-your-user-interface-with-polaris-starter-files`